src/main.ts

```diff
@@ -0,0 +1,16 @@
+export { BooleanSBVR }     from './types/BooleanSBVR'
```

Comment:
Can you rename this file index.ts (it's a more standard name for the entry point and has special features as such, eg `require('./src')` will actually require `./src/index.*` if it's a directory)